### DISCUSSION TITLE
Migrate to GitHub discussions beta from Spectrum?

### DISCUSSION THREAD

**Discussion by tom-sherman** [ariakit/ariakit#734] [Questions] [answered]
It looks as though it's [possible to apply](https://github.community/t/can-one-apply-for-github-discussions-beta/121120/16) to join the GitHub Discussions beta.

Examples:

https://github.com/githubevents/satellite2020/discussions

https://github.com/vercel/vercel/discussions

[a comment by an automated account is not shown]

**Comment by diegohaz** [comment]
Thanks Tom! I just commented on that thread requesting access.

**Comment by diegohaz** [comment, marked as answer]
YAAY!

https://github.com/reakit/reakit/discussions

**Reply by tom-sherman** [reply]
🎉🎊
